app/views/schools/on_boarding/profiles/show.html.erb

```diff
@@ -25,7 +25,7 @@
     <dl class="govuk-summary-list">
       <%= summary_row 'Fees', @profile.fees, edit_schools_on_boarding_fees_path %>
       <%= summary_row 'DBS check required', @profile.dbs_check, edit_schools_on_boarding_dbs_requirement_path %>
-      <%= summary_row 'Individual requirements', @profile.individual_requirements, edit_schools_on_boarding_candidate_requirements_choice_path %>
+      <%= summary_row 'Individual requirements', @profile.individual_requirements, edit_schools_on_boarding_candidate_requirements_choice_path, display_as_list: true %>
```

Comment:
Rather than teach summary row to format content, its better to provide it with already formatted content

```erb
<%= summary_row 'Individual requirements', split_to_list(@profile.individual_requirements), edit_schools_on_boarding_candidate_requirements_choice_path %>
```